tests/Library.So/Test.mk

```diff
@@ -95,8 +95,8 @@ clean-destdir: clean-destdir.log
 	GREP -qFx 'rm -f ./libbar.so-bar.d' <$<
 
 all-enable-dynamic-libs: all-enable-dynamic-libs.log
-    # Configuring --enable-dynamic enables compliation of dynamic libraries.
+	# Configuring --enable-dynamic enables compliation of dynamic libraries.
```

Comment:
NIT: Two more s/compliation/compilation/
grep for 'compliation'

Reply:
Uh, my vim settings are wrong and I had spellchecker off. Thanks for spotting that